### PR TITLE
README update

### PR DESCRIPTION
**Pull request by jacknagz**
Additional edits to the README.

* Modify the layout to look prettier
* Point our Getting Started to the Gitbook documentation
* Add UI screenshots
* Ignore Mac DS_Store files

* Verified in the Github UI

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,69 +1,69 @@
...
-   - Near the end of the deploy command, you'll be prompted for first/last name and email
-   - You will get an email from **no-reply@verificationemail.com** with your temporary password. If you don't see it, be sure to check your spam folder.
-8. Sign in to Panther! The URL is listed in the welcome email and also printed at the end of the deploy command.
-   - WARNING: By default, Panther generates a self-signed certificate, which will cause most browsers to present a warning page.
-   - If you see a "502 Bad Gateway" error, wait a few minutes and refresh the page
-9. [Onboard your AWS account(s)](https://docs.runpanther.io/quick-start) in your Panther deployment!
+## Deployment
 
-## Development
+Panther is deployed from within your AWS account. Follow our [Quick Start guide here](https://docs.runpanther.io/quick-start) to quickly get up and running!
```
Comment (austinbyers) [inline README.md]: Nit: Can we simplify this to a single sentence:

Follow our [Quick Start](...) guide to deploy Panther to your AWS account!

("from within" makes it sound like you login to your AWS account console and deploy it from there)

File: README.md [hunk 2]
```diff
@@ -1,69 +1,69 @@
...
+<img src="docs/img/compliance-overview.png" alt="Compliance Overview"/>
+<p align="center"><i>Compliance Overview</i></p>
+
+<img src="docs/img/rules-editor.png" alt="Rules Editor"/>
+<p align="center"><i>Rules Editor</i></p>
+
+<img src="docs/img/resource-viewer.png" alt="Resource Viewer"/>
+<p align="center"><i>Resource Viewer</i></p>
+
+## Development
```
Comment (austinbyers) [inline README.md]: With the "getting started" section moved out of the README, I think this section (development + testing) should move out as well.

The `mage` commands listed here won't even work until you setup the repo and install mage, for example

File: README.md [hunk 3]
```diff
@@ -1,69 +1,69 @@
...
 
-## Development
+Panther is deployed from within your AWS account. Follow our [Quick Start guide here](https://docs.runpanther.io/quick-start) to quickly get up and running!
 
-Since the majority of Panther is written in Go, we follow the [standard Go project layout](https://github.com/golang-standards/project-layout).
+## Screenshots
 
-Run `mage` to see the list of available commands (`-v` for verbose mode). You can easily chain `mage` commands together, for example:
+<img src="docs/img/compliance-overview.png" alt="Compliance Overview"/>
+<p align="center"><i>Compliance Overview</i></p>
```
Comment (austinbyers) [inline README.md]: Can we add some extra space between images? It's not clear which image the caption applies to:

Screen Shot 2020-01-22 at 10 34 10 AM

Comment (jacknagz) [reply README.md]: Sure, maybe a line break would help

File: README.md (hunk 2 above)
Comment (jacknagz) [reply README.md]: Sounds good, removing

**Pull request merged**